clusters/dev/01/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
We don't have any cluster manifests at the moment, but we may add some cluster level manifests in the future.